src/custom/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -0,0 +1,124 @@
+// import { Trade , TradeType } from '@uniswap/sdk'
```

Comment:
I think this file is technically a "Mod" file, but it doesn't use our procedure for extending the functionality.